test/extensions/filters/http/file_system_buffer/BUILD

```diff
@@ -34,3 +34,16 @@ envoy_cc_test(
         "//test/mocks/upstream:upstream_mocks",
     ],
 )
+
+envoy_cc_test(
```

Comment:
Sorry I should have caught this before, but for concrete filters, these should be envoy_extension_cc_test and the filter code itself should be extension targets. If there is some reason you are doing this switch in a future PR can you add TODOs in various places?

Reply:
Added TODO (it is indeed done in the next PR - the extension doesn't truly have a name until then. Maybe that would be fine, I don't really know what the purpose of the extension name tag is in extension_cc_tests.)